lib/config/definitions.ts

```diff
@@ -447,6 +447,14 @@ const options: RenovateOptions[] = [
     type: 'string',
     default: null,
   },
+  {
+    name: 'prCommitsPerRunLimit',
+    description:
+      'Set a maximum number of commits per Renovate run. Default is no limit.',
+    stage: 'global',
+    type: 'integer',
+    default: null,
```

Comment:
Would using `null` for "no limit" be better or `0` for no limit?

Reply:
I have changed it to zero to illustrate it makes no difference